ui/packages/consul-peerings/app/templates/dc/peers/index.hbs

```diff
@@ -60,8 +65,56 @@ as |sort filters items|}}
         />
       {{/if}}
 
+    </BlockSlot>
+    <BlockSlot @name="actions">
+
+
+      <ModalDialog
+        @aria={{hash
+          label="Add peer connection"
+        }}
+        @onclose={{optional this.form.reset}}
```

Comment:
won't the form state be reset when the `modal.opened` conditional is falsy?

Reply:
I'm not certain, but I think what @LevelbossMike is asking is whether the form will reset automatically when the `modal.opened` conditional changes, and therefore whether this is necessary to specify?